main.yml

```diff
@@ -424,25 +424,41 @@ staging:
   frontend:
     sub_apps:
       - id: ruledev
-      - id: subscriptions
   top_level: true
 
 subscriptions:
-  title: Subscription Reporting
+  title: Subscription Watch
   api:
     versions:
       - v1
-    tags:
-      - value: "experimental"
-        title: "Experimental API"
     isBeta: true
   channel: '#subscriptions'
   deployment_repo: https://github.com/RedHatInsights/curiosity-frontend-build
+  top_level: true
   frontend:
     paths:
-      - /staging/subscriptions
+      - /subscriptions
+    sub_apps:
+      - id: subscriptions-rhel
```

Comment:
Confirmed, looks like we're going to flip the ID over to `rhel-sw`. We came up with a naming scheme since we're also going to have an `Openshift` left nav item, so if we can re-ID towards
```
- id: rhel-sw
```

That would help us out